### PR TITLE
Update SEEEDSTUDIO_WIO_E5.md- bind button

### PR DESCRIPTION
**Pull request by 4O6FPV**
clarified what the button does for each device e5 and how to enter bind mode from the button. I know this is probably listed elsewhere but doesn't hurt to have it here I think?

### CODE REVIEW

File: docs/SEEEDSTUDIO_WIO_E5.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Connections (name in respect to board print-ons):
 - debug: A3
 - led green: SDA (solder a green LED with resistor > 300 Ohm to GND, you can use pad SCL as intermediate post)
 - led red: on-board
-- button: on-board (BOOT button)
+- bind button: on-board (BOOT button, hold for 4 seconds to initiate bind mode)
```
Comment (olliw42) [inline docs/SEEEDSTUDIO_WIO_E5.md:23]: maybe "bind button: BOOT button (hold for..."
dito below

File: docs/SEEEDSTUDIO_WIO_E5.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Connections (name in respect to board print-ons):
 - debug: Tx, and on-board USB plug
 - led green: SDA (solder a green LED with resistor > 300 Ohm to GND, you can use pad SCL as intermediate post)
 - led red: on-board
-- button: on-board (BOOT button)
+- bind button: on-board (BOOT button, hold for 4 seconds to initiate bind mode)
```
Comment (olliw42) [inline docs/SEEEDSTUDIO_WIO_E5.md:43]: dito

**Comment by olliw42** [comment]
LGTM
let's see what @jlpoltrack thinks

**Comment by jlpoltrack** [comment]
Looks good to me as well - merging, many thanks.

**Pull request merged**